README.md

```diff
@@ -24,27 +24,26 @@ To run this sample:
 1. Clone this repository and navigate to the new directory.
 
     ```bash
-    $ git clone git@github.com:okta/samples-js-angular-1.git && cd samples-js-angular-1
+    git clone git@github.com:okta/samples-js-angular-1.git
+    cd samples-js-angular-1
     ```
 
 2. Install the required dependencies.
 
     ```bash
-    [samples-js-angular-1]$ npm install
+    npm install
     ```
 
 3. Run the mock Okta server. If you're interested in testing this sample against your own Okta org, follow the steps in the [Express Quick Start](https://github.com/okta/samples-nodejs-express-4#quick-start).
 
     ```bash
-    # Starts the mock-okta server at http://127.0.0.1:7777
```

Comment:
Any reason why we are removing these comments? Until we remove `mock-okta`, these are pretty helpful to the reader.

Reply:
Comments using `#` are a bash-only thing. We could move out out into the text of the readme.

How often does the user need to know what port the mock-okta server is running on? Since the sample is already configured to point with it, I assumed there's not much value in knowing it's on `7777`.